src/providers/node/mod.rs

```diff
@@ -179,6 +192,18 @@ impl NodeProvider {
             return Ok(Some(format!("node {}/index.js", output_path)));
         }
 
+        if Turborepo::is_turborepo(app) {
+            let turbo_cfg = Turborepo::get_config(app)?;
+            let app_name = Turborepo::get_app_name(env);
+
+            if let Some(start_pipeline) = Turborepo::get_start_cmd(&turbo_cfg) {
+                return Ok(Some(start_pipeline));
+            } else if let Some(name) = app_name {
+                return Ok(Some(format!("npx turbo run {}:start", name)));
```

Comment:
We should check for which package manager to use

Reply:
+1. There are a few places it should use the package manager.